package/src/editor/api/index.js

```diff
@@ -301,5 +305,29 @@ export const EditorApi = Object.extend(
 
     this.commonPageConfigurationTabs.apply(view);
     return view;
+  },
+
+  /**
+   * Ensures that Editor only runs on modern browsers.
+   *
+   * @param {function} start
+   *  A callback function to start the app.
+  */
+  ensureBrowserSupport: function(start) {
+    if (this.entryType.isBrowserSupported) {
+      const isBrowserSupported = this.entryType.isBrowserSupported();
+      const browserNotSupportedView = new this.entryType.browserNotSupportedView;
```

Comment:
Missing "()" when invoking a constructor. This should show a linter warning in your editor. TIL this is not a syntax error.

Also, we could consider only creating the view when it is actually needed.